gprofiler/spark/sampler.py

```diff
@@ -41,13 +33,10 @@ def __init__(
         storage_dir: str,
         api_client: Optional[APIClient] = None,
     ):
-        self._master_address: Optional[str] = None
-        self._spark_mode: Optional[str] = None
         self._collection_thread: Optional[Thread] = None
         self._sample_period = sample_period
         # not the same instance as GProfiler._stop_event
         self._stop_event = Event()
-        self._spark_sampler: Optional[SparkCollector] = None
         self._stop_collection = False
```

Comment:
Not used - remove?

Reply:
a1bf1d0e7580c5afd37133462d23778deeeff816